website/docs/reference/node-selection/methods.md

```diff
@@ -352,3 +352,16 @@ dbt list --select version:none        # models that are *not* versioned
 ```
 
 </VersionBlock>
+
+<VersionBlock firstVersion="1.6">
+
+The `semantic_model` method selects [semantic models](/docs/build/semantic-models).
+
+```bash
+
+dbt ls --resource-type semantic_model # lists the semantic model resource type
+dbt ls -s +semantic_model:number # lists the quantity of semantic models 
+
```

Comment:
For consistency with the rest of this page:
1. fix the accuracy of the comment
1. showing selection of all nodes using this method
1. updating `ls` to be `list`, `-s` to be `--select`
1. removing leading and trailing newlines
1. adjusting comment indentation
1. removing `dbt list --resource-type semantic_model   # lists the semantic model resource type` in favor of docs on `website/docs/reference/commands/list.md`

```suggestion
dbt list --select +semantic_model:*       # lists all semantic models 
dbt list --select +semantic_model: orders  # lists all resources upstream of your semantic model named "orders"
```

⚠️ Caveat: I didn't actually try out this syntax! Someone should confirm it behaves how it is described.

i.e., after DM'ing with @dave-connors-3,  `+semantic_model:number` doesn't "lists the quantity of semantic models",  but rather selects a particular semantic model based off its identifier.

Reply:
by including the plus in the first example, you would get the full list of parent nodes too! i would remove the plus from the first example to match the comment.

Reply:
I have confirmed the behavior on my local project using `dbt=1.7.0-b2`!

Reply:
> by including the plus in the first example, you would get the full list of parent nodes too! i would remove the plus from the first example to match the comment.

👍 Updated the example to remove the [plus operator](https://docs.getdbt.com/reference/node-selection/graph-operators#the-plus-operator)